snapper-samples/build.gradle

```diff
@@ -22,5 +22,5 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:23.0.1'
```

Comment:
No? The samples are illustrative, but aren't for inclusion and the library part doesn't depend on the samples.

However, back to the original question, there is a leftover dependency in snapper's `build.gradle` on AppCompat which I'll remove, but for reference it works fine with different versions.